src/test/scala/cromwell/CopyWorkflowOutputsSpec.scala

```diff
@@ -0,0 +1,43 @@
+package cromwell
+
+import java.nio.file.{Files, Paths}
+
+import akka.testkit.EventFilter
+import cromwell.util.SampleWdl
+import org.scalatest.prop.TableDrivenPropertyChecks._
+import org.scalatest.prop.Tables.Table
+
+
+class CopyWorkflowOutputsSpec extends CromwellTestkitSpec {
+
+  "CopyWorkflowOutputs" should {
+    "copy workflow outputs" in {
+      val workflowOutputsPath = "copy-workflow-outputs"
+
+      val tmpDir = Files.createTempDirectory(workflowOutputsPath).toAbsolutePath
+
+      val outputs = Table(
+        ("call", "file"),
+        ("call-A", "out"),
+        ("call-A", "out2"),
+        ("call-B", "out"),
+        ("call-B", "out2")
+      )
+
+      val workflowId = runWdlAndAssertOutputs(
+        sampleWdl = SampleWdl.WorkflowOutputsWithFiles,
```

Comment:
This WDL lists every output as a workflow output. For the sake of the test, can we have an output which isn't a *workflow* output and confirm that it is *not* copied?